### DISCUSSION TITLE
What is the future of the multiple pages to show starred repositories?

### DISCUSSION THREAD

**Discussion by llrs** [community/community#8657] [Other Feature Feedback, Questions, & Ideas] [answered]
Many thanks for adding lists of starred repos!
I was using https://github.com/stars/ to find starred repositories but currently it doesn't show the lists of repositories. However, when using lists of repositories I cannot search on these specific repositories (or to find the right repository I had in mind). Hope this will come soon, otherwise they are currently useless, when there is more than 1 page of repositories, in my opinion.

What is the future of https://github.com/stars/ and `https://github.com/USERNAME?tab=stars` ?

**Comment by brianlovin** [comment, marked as answer]
Thanks for the questions @llrs — right now it's a known issue that Lists don't appear on the `/stars` page. For now, I would recommend using the `/USERNAME?tab=stars` page. Better search, sort, and filter controls are being requested by a lot of people, so we'll see how we can make improvements there in the future!

**Reply by llrs** [reply]
Many thanks for the quick reply. Great to hear you are considering adding more filter and search tools for starred repos. Looking forward to this
